text/0000-non-exhaustive.md

```diff
@@ -433,6 +437,41 @@ match it like:
 let Unit { .. } = unit;
 ```
 
+To users of this crate, this will act exactly as if the struct were defined as:
+
+```
+#[non_exhaustive]
+pub struct Unit {}
```

Comment:
Well, this seems to be equivalent to https://github.com/rust-lang/rfcs/pull/2008/commits/3d568896c29e3071a7582533418e02c6f32fa085#r135395315, but I'd still prefer to use the same wording for tuple and unit structs.